pyanaconda/payload/__init__.py

```diff
@@ -779,7 +779,7 @@ def _find_mounted_additional_repos(self):
     def _unmount_source_directory(self, mount_point):
```

Comment:
The commit comment shouldn't be ``Use the function resolve_device from payload utils`` but instead ``Add new function resolve_device to payload utils``

Reply:
Fixed.